Cargo.toml

```diff
@@ -1,26 +1,21 @@
 [package]
 name = "tree-sitter-jsonnet"
 description = "jsonnet grammar for the tree-sitter parsing library"
-version = "0.0.1"
+version = "0.0.2"
 keywords = ["incremental", "parsing", "jsonnet"]
 categories = ["parsing", "text-editors"]
 repository = "https://github.com/tree-sitter/tree-sitter-jsonnet"
 edition = "2018"
 license = "MIT"
 
 build = "bindings/rust/build.rs"
-include = [
-  "bindings/rust/*",
-  "grammar.js",
-  "queries/*",
-  "src/*",
-]
+include = ["bindings/rust/*", "grammar.js", "queries/*", "src/*"]
 
 [lib]
 path = "bindings/rust/lib.rs"
 
 [dependencies]
-tree-sitter = "0.22.6"
+tree-sitter = "^0.24.5"
```

Comment:
This is not more lenient than the previous range. The meaning of 0.22.6 is `0.22.6 <= v < 0.23`. The meaning of `^0.24.5` is `0.24.5 <= v < 0.25`. So the ranges do not have overlap.

What is the reason for bumping the ranges here? Is it for the syntax highlighter in the monorepo? 

Over there, we need all grammars to be compiled with the same version of tree-sitter. Right now, the Cargo.toml for the workspace in the monorepo requires `0.22.6 <= v < 0.23`.

https://sourcegraph.sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/docker-images/syntax-highlighter/Cargo.toml?L50